docker-compose.yml

```diff
@@ -1,22 +1,31 @@
 services:
   actual_server:
+    ## Common tags: latest, edge (ADVANCED ONLY), lastest-alpine, edge-alpine (ADVANCED ONLY), specific version (e.g. 24.8.0, 24.8.0-alpine)
     image: docker.io/actualbudget/actual-server:latest
     ports:
-      # This line makes Actual available at port 5006 of the device you run the server on,
-      # i.e. http://localhost:5006. You can change the first number to change the port, if you want.
+      ## This line makes Actual available at port 5006 of the device you run the server on,
+      ## i.e. http://localhost:5006. You can change the first number to change the port, if you want.
       - '5006:5006'
     environment:
-      # Uncomment any of the lines below to set configuration options.
-      # - ACTUAL_HTTPS_KEY=/data/selfhost.key
-      # - ACTUAL_HTTPS_CERT=/data/selfhost.crt
+      ## Uncomment any of the lines below to set configuration options.
+      # - ACTUAL_HTTPS_KEY=/data/certs/selfhost.key
+      # - ACTUAL_HTTPS_CERT=/data/certs/selfhost.crt
       # - ACTUAL_PORT=5006
```

Comment:
I wonder if this should have a comment pointing out that if you change it from 5006, you need to update line 8 to match?